docs/units/quantity.rst

```diff
@@ -517,6 +517,44 @@ It can also be used for in-place conversion::
     >>> a  # doctest: +FLOAT_CMP
     array([-100.,  100.,  200.,  300.,  400.])
 
+
+The `numpy.dtype` of a Quantity
+===============================
+
+|Quantity| subclasses `numpy.ndarray` and similarly accepts a ``dtype``
+argument.
+
+    >>> q = u.Quantity(1.0, dtype=np.float32)
+    >>> q.dtype
+    dtype('float32')
+
+Like for `numpy.ndarray`, ``dtype`` does not have to be specified, in which
+case the data is inspected to find the best ``dtype``. For `numpy` this means
+integers remain integers, while |Quantity| instead upcasts integers to floats.
+
+    >>> v = np.array(1)
+    >>> np.issubdtype(v.dtype, np.integer)
+    True
+
+    >>> q = u.Quantity(1)
+    >>> np.issubdtype(q.dtype, np.integer)
+    False
+
+|Quantity| promotes integer to floating types because it has a different
+default value for ``dtype`` than `numpy` -- `numpy.floating` versus `None`.
```

Comment:
And this one too - `np.inexact`. Though I wonder how useful it is to mention the new default explicitly (see also for the what's-new entry).

Reply:
I agree that it's probably not worth mentioning in the whatsnew, but here maybe because while it's useful here, but deprecated and raises a warning in numpy